src/Http/Client/Adapter/Stream.php

```diff
@@ -293,7 +293,7 @@ protected function _send(Request $request)
      */
     protected function _open($url)
     {
-        set_error_handler([$this, '_connectionErrorHandler']);
+        set_error_handler([$this, 'connectionErrorHandler']);
```

Comment:
That would make a lot of sense. I'll wait and see what others think before I do that.